test/environment_test.rb

```diff
@@ -461,7 +461,7 @@
     end
   end
 
-  describe '#code_reloading?' do
+  describe '#code_reloading?', engine: :mri do
```

Comment:
@andrewhr Thats for sure! Although, I think it may be quite complex to achieve, things like [Drip](https://github.com/flatland/drip) are not too good, process forking is too expensive within jRuby. I don't know, I would like something like Hot Deploy™